ElectronApp/src/renderer/App.tsx

```diff
@@ -0,0 +1,22 @@
+import { Outlet, Link } from 'react-router-dom';
+
+export default function App() {
+  return (
+    <div>
+      <h1>ForestGEO App</h1>
+      <nav
+        style={{
```

Comment:
Should this style be in App.css?

Reply:
Good call. I based this off of the react-router tutorial which probably had it all in App.tsx for simplicity's sake, but you're right it should be in App.css.